src/python/grpcio/grpc/experimental/aio/_channel.py

```diff
@@ -238,13 +251,15 @@ def __init__(self, target: str, options: ChannelArgumentType,
 
             invalid_interceptors = set(interceptors) - set(
                 self._unary_unary_interceptors) - set(
-                    self._unary_stream_interceptors)
+                    self._unary_stream_interceptors) - set(
+                        self._stream_unary_interceptors)
 
             if invalid_interceptors:
                 raise ValueError(
                     "Interceptor must be "+\
                     "UnaryUnaryClientInterceptors or "+\
-                    "UnaryStreamClientInterceptors. The following are invalid: {}"\
+                    "UnaryUnaryClientInterceptors or "+\
+                    "StreamUnaryClientInterceptors. The following are invalid: {}"\
```

Comment:
nit: We can programmatically get the name of these three classes.
```python
tuple(interceptor_class.__name__ for _, interceptor_class in attrs_and_interceptor_classes)
```

Reply:
I guess that would prevent automatically to have typos, Ill do it.